com.skype.Client.appdata.xml

```diff
@@ -21,6 +21,6 @@
   <icon type="remote" height="64" width="64">https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w64</icon>
   <icon type="remote" height="128" width="128">https://lh3.googleusercontent.com/QfAEt_ya6-n8w_TD9-PsghFC2DMSO7fLGNZB4cQ3RtbBbHFkXJE_gxOc3l32-j6LXg=w128</icon>
   <releases>
-    <release date="2018-01-07" version="8.13.76.8"/>
+    <release date="2018-01-26" version="8.15.76.1"/>
```

Comment:
I believe that this should be a new entry instead of superseding the old one.

Reply:
Without changelogs I don't think it really matters. We are largely just using it to show the current versions in the UI.

Reply:
I agree, there is no point in showing older releases without change log, besides, only the latest version could be installed anyway